doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -238,7 +238,11 @@ Non-resilient applications are sensitive to platform impairments on Compute like
 <p align="center"><b>Table 3-1:</b> Categories of applications, requirements for scheduling pods and Kubernetes features</p>
 
 Kubernetes clusters using above enhancements can implement worker nodes with "bare metal" servers (running Container Runtime in Linux host Operating System) or with virtual machines (VMs, on hypervisor).
-When running in VMs, the following requirements and configuration are needed for performance optimization: CPU Manager for vCPUs in VM. Huge pages enabled in hypervisor and mapped into VM. Hardware Topology Management with hypervisors enabled for NUMA. Node Feature Discovery and Device Plugin Framework for CPU instructions and devices exposed through hypervisor.
+When running in VMs, the following list of configurations shows what is needed for non-resilient applications:
```

Comment:
Why "non-resilient"? The elements in the bullets are related to performance.

Reply:
> Why "non-resilient"? The elements in the bullets are related to performance.

@pgoyal01 : That section is already called "Scheduling Pods with Non-resilient Applications" and starts with "Non-resilient applications are sensitive to platform impairments..." which is more precise description of target apps. With more application resilience, we could get to good levels of "performance" without some of those enhancements.